examples/libs/tezos/basic-tx/taquito-injection/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "taquito-injection",
```

Comment:
can we rename to `taquito-basic-tx` and remove the`taquito-injection` folder? see how the other folders are structured for example.